### ISSUE TITLE
StyleObj is no longer part of react-native

### ISSUE THREAD

**Issue by maxammann** [vonovak/react-navigation-header-buttons#31]
There are several imports of StyleObj. flow throws errors because this is no longer part of react-native: https://stackoverflow.com/a/49799497

flow error:
```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ node_modules/react-navigation-header-buttons/src/HeaderButtons.js:8:15

Cannot import StyleObj because there is no StyleObj export in react-native/Libraries/StyleSheet/StyleSheetTypes.

      5│ import { HeaderButton, type HeaderButtonProps, type VisibleButtonProps } from './HeaderButton';
      6│ import { StyleSheet, Platform, View, Text } from 'react-native';
      7│ import { OverflowButton, type OverflowButtonProps, IS_IOS } from './OverflowButton';
      8│ import type { StyleObj } from 'react-native/Libraries/StyleSheet/StyleSheetTypes';
      9│
     10│ const textTransformer = (label: string) =>
     11│   IS_IOS ? label.charAt(0).toUpperCase() + label.substr(1) : label.toUpperCase();
```

**Comment by vonovak** [comment]
fixed in 2.2.1

**Comment by maxammann** [comment]
woop woop, thanks :)
